bedrock/events/models.py

```diff
@@ -14,40 +14,58 @@
 from bedrock.events.countries import country_to_continent
 
 
+def calendar_id_from_google_url(feed_url):
+    return feed_url.split('/')[5]
+
+
+def calendar_url_for_event(event, calendar_id):
+    template = 'https://www.google.com/calendar/embed?showCalendars=0&mode=week&wkst=1&' \
+               'src={calendar_id}&ctz=America/Los_Angeles&' \
```

Comment:
> Why would we need to escape ampersands? IIRC they only need to be escaped if you want to actually display an ampersand in the page.

You can usually get away without escaping ampersands in URLs, but it's safer to default to escaping them. See the top answer to http://stackoverflow.com/questions/19441750/do-ampersands-still-need-to-be-encoded-in-urls-in-html5 for a more detailed explanation.